packages/flutter_tools/gradle/aar_init_script.gradle

```diff
@@ -76,7 +103,7 @@ void configurePlugin(Project project, String outputDir) {
         // A plugin doesn't support the Android platform when this property isn't defined in the plugin.
         return
     }
-    configureProject(project, outputDir)
+    configureProject(project, outputDir, false)
```

Comment:
For module projects, it should use '--builder-number' as the version number; therefore, I passed the project type to the 'configureProject' function.